Source/WebKit/UIProcess/BrowsingContextGroup.cpp

```diff
@@ -187,10 +189,10 @@ RefPtr<RemotePageProxy> BrowsingContextGroup::takeRemotePageInProcessForProvisio
     auto it = m_remotePages.find(page);
     if (it == m_remotePages.end())
         return nullptr;
-    auto* remotePage = remotePageInProcess(page, process);
+    RefPtr remotePage = remotePageInProcess(page, process);
     if (!remotePage)
         return nullptr;
-    return it->value.take(remotePage);
+    return it->value.take(remotePage.get());
```

Comment:
I'm a bit unsure for this one, it seems it's only used as a key for the hash set... how do we handle that normally?

Reply:
I guess this is the same answer as below for BrowsingContextGroup::transitionProvisionalPageToRemotePage.